lib/js-file.js

```diff
@@ -134,6 +136,50 @@ JsFile.prototype = {
     },
 
     /**
+     * Sets whitespace before specified token.
+     *
+     * @param {Object} token
+     * @param {String} whitespace
+     */
+    setWhitespaceBefore: function(token, whitespace) {
```

Comment:
Tried to decompose, but it looked ugly. I've simplified a bit and added comments instead.